frontend/src/GameReports.tsx

```diff
@@ -475,11 +479,30 @@ function FixedWidthText({ width, children }: FixedWidthTextProps) {
     );
 }
 
-function WrappedText({ children }: ContainerProps) {
+function CommentText({ children }: ContainerProps) {
     return (
-        <Box width="400px" whiteSpace="wrap">
-            {children}
-        </Box>
+        <Tooltip
+            title={<Box maxWidth="300px">{children}</Box>}
+            enterTouchDelay={0}
+            size="sm"
+        >
+            <Box
+                sx={{
+                    display: "flex",
+                    alignItems: "center",
+                    justifyContent: "center",
+                    cursor: "pointer",
+                    color: "primary.500",
+                    px: "5px",
+                }}
+            >
+                <ViewIcon
+                    color="primary"
+                    sx={{ width: "1.5em", height: "1.5em", pr: "5px" }}
+                />
+                Read comments
```

Comment:
Maybe this should be "View comments" or even just "View"?

Reply:
I'm ambivalent! I think "Read comments" and "View comments" work equally well, and I have a slight preference for either over just "View"

Reply:
Okay cool cool, I updated to "View comments," that one wins for me